team-validator.js

```diff
@@ -9,7 +9,7 @@
 
 'use strict';
 
-let TeamValidator = module.exports = getValidator;
+let TeamValidator = exports.TeamValidator = getValidator;
```

Comment:
So I could export both `TeamValidator` and `TeamValidatorMonitor`, though I'm assuming I'm supposed to be doing `TeamValidator.TeamValidatorMonitor` or something instead